pages/index.js

```diff
@@ -0,0 +1,39 @@
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
```

Comment:
I'm just curious do we have any plan to add the [absolute imports or module path aliases](https://nextjs.org/docs/advanced-features/module-path-aliases)? so later when the project grow bigger and there's a lot of stylesheets to import we can use something like `styles/**` or `@/styles/**`

Reply:
@carryall , I think we might add it, but IMHO we can skip it in this initialization phase for now and can look into it during https://github.com/tyrro/react-survey/issues/9 , What do you think?

Reply:
works for me 🙌🏼